### ISSUE TITLE
Error on logits sign (paddle.py)

### ISSUE THREAD

**Issue by QuentinJGMace** [SegoleneMartin/PADDLE#3]
I think there is a mistake in the way logits are computed in paddle.py file

This is the code right now
` logits = (- samples.matmul(self.w.transpose(1, 2)) - 1 / 2 * (self.w**2).sum(2).view(n_tasks, 1, -1) - 1 / 2 * (samples**2).sum(2).view(n_tasks, -1, 1))`
`return - logits`

I think it should be 
`logits = ( samples.matmul(self.w.transpose(1, 2)) - 1 / 2 * (self.w**2).sum(2).view(n_tasks, 1, -1) - 1 / 2 * (samples**2).sum(2).view(n_tasks, -1, 1)) `
`return logits`

**Comment by SegoleneMartin** [comment]
Yes sorry about that and thanks for pointing it out ! I fixed it.
